packages/clerk-js/src/ui/elements/Modal.tsx

```diff
@@ -54,6 +54,7 @@ export const Modal = withFloatingTree((props: ModalProps) => {
       nodeId={nodeId}
       context={context}
       isOpen={isOpen}
+      outsideElementsInert
```

Comment:
This was needed to work with Drawer usage within the Modal. Without it, inert attributes would not be properly removed after closing the Modal. Currently folks are not interacting with elements outside of the modals so there should be no difference in experience.